logstash-core/lib/logstash/api/lib/app/modules/stats.rb

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+require "app"
+
+module LogStash::Api
+  class Stats < BaseApp
+
+    helpers AppHelpers
+
+    # return hot threads information
+    get "/jvm" do
+      command = factory.build(:memory_command)
+      jvm_payload = {
+        :timestamp => command.started_at,
+        :uptime_in_millis => command.uptime,
```

Comment:
this is to match with the expected ES format, I understand your concerns, but we try to match as close as possible with them.